scripts/ci/batch_unit_mutation_lint.sh

```
@@ -0,0 +1,226 @@
+    echo "🎉 Production environment started successfully!"
+}
+
+run_make_with_dind() {
+    local target=$1
+    local description=$2
+    local website_dir=$3
+
+    echo "🔧 Ensuring network and dev service via Make"
+    setup_docker_network
+    # Use compose health, not host wait-on, by overriding NEXT_DEV_CMD
+    if ! make start NEXT_DEV_CMD='$(DOCKER_COMPOSE) $(DOCKER_COMPOSE_DEV_FILE) up -d --wait dev'; then
```

Shellcheck (0.10.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)